docs/ops/generation/Multinomial_13.md

```diff
@@ -0,0 +1,151 @@
+  * **Range of values**: `true`, `false`
+      * ``true`` - class indices can be sampled multiple times.
+      * ``false`` - class indices will not repeat in the output and size of *input*'s ``class_size`` dimension is required to be larger or equal to *num_samples* value.
+  * **Type**: `bool`
+  * **Required**: *Yes*
+
+* ``log_probs``
+
+  * **Description**: allows to control whether *inputs* should be treated as probabilities or unnormalized log probabilities.
+  * **Range of values**: `true`, `false`
+      * ``true`` - set values in *inputs* are unnormalized log probabilities that can be any real number.
+      * ``false`` - probabilities in *inputs* are expected to be non-negative, finite and have non-zero-sum.
+  * **Type**: `bool`
+  * **Required**: *Yes*
+
+* ``global_seed``
+
+  * **Description**: global seed value.
+  * **Range of values**: positive integers
+  * **Type**: `int`
+  * **Default value**: 0
+  * **Required**: *Yes*
+
+* ``op_seed``
+
+  * **Description**: operational seed value.
+  * **Range of values**: positive integers
+  * **Type**: `int`
+  * **Default value**: 0
+  * **Required**: *Yes*
+
+**Inputs**:
+
+*   **1**: ``input`` - 1D or 2D tensor of type `T_IN` and shape `[class_size]` or `[batch_size, class_size]` containing probabilities. Allowed values depend on the *log_probs* attribute and are internally normalized to have values in the range of `[0, 1]`, sum of all probabilities in the given batch is equal to 1. **Required.**
+
+*   **2**: ``num_samples`` - scalar or 1D tensor with 1 element of type `T_SAMPLES` specifying the number of samples to draw from Multinomial distribution. **Required.**
+
+**Outputs**:
+
+* **1**:  ``output`` A tensor with type specified by the attribute *output_type* and shape depending on *input* and *num_samples*, either ``[num_samples]`` or ``[batch_size, num_samples]``.
+
+**Types**
+
+* **T_IN**: any supported floating-point type.
+* **T_SAMPLES**: any supported integer type.
+
+
+*Example 1: 1D input tensor.*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... name="Multinomial" type="Multinomial">
+        <data output_type="f32" replacement="true", log_probs="false" global_seed="234" op_seed="148"/>
+        <input>
+            <port id="0" precision="FP32">  < !-- shape value: [0.1, 0.5, 0.4] -->
+                <dim>3</dim>
+            </port>
+            <port id="1" precision="I32"/> < !-- num_samples value: 5 -->
+        </input>
+        <output>
+            <port id="3" precision="FP32" names="Multinomial:0">
+                <dim>5</dim>
+            </port>
+        </output>
+    </layer>
+
+*Example 2: 2D input tensor.*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... name="Multinomial" type="Multinomial">
+        <data output_type="f32" replacement="true", log_probs="true" global_seed="234" op_seed="148"/>
+        <input>
+            <port id="0" precision="FP32">  < !-- shape value: [50, 1, 21] -->
```

Comment:
```suggestion
            <port id="0" precision="FP32">  < !-- shape value: [50, 1, 21, ...] -->
```
As shape has 48 elements but here are only three.
What about examples of output values? Add them as example.